src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -372,11 +359,10 @@ internal IEnumerable<V> Search(string query, int minResultsForTolerantSearch = 0
             var orderedSearchDict = searchDict.OrderByDescending(x => x.Value);
 
             var searchResults = orderedSearchDict.Select(x => x.Key);
-            if (TruncateSearchResults)
-            {
-                searchResults = searchResults.Take(20);
-            }
 
+            // return only the top 30 search results
```

Comment:
Not that I'm against it, but why the change from 20 to 30 here?

Reply:
Just saw the conversation in the main comments, it didn't show up in the file comparison view. This is fine with me.